qa-pipelines/tasks/cf-upgrade.sh

```diff
@@ -52,33 +52,38 @@ set_uaa_sizing_params # Adds uaa sizing params to HELM_PARAMS
 echo UAA customization ...
 echo "${HELM_PARAMS[@]}" | sed 's/kube\.registry\.password=[^[:space:]]*/kube.registry.password=<REDACTED>/g'
 
-helm upgrade uaa ${CAP_DIRECTORY}/helm/uaa/ \
-    --namespace "${UAA_NAMESPACE}" \
-    --recreate-pods \
-    --timeout 3600 \
-    --wait \
-    "${HELM_PARAMS[@]}"
-
-# Wait for UAA release
-wait_for_release uaa
+if [ "${EMBEDDED_UAA:-false}" != "true" ]; then
+  helm upgrade uaa ${CAP_DIRECTORY}/helm/uaa/ \
+      --namespace "${UAA_NAMESPACE}" \
+      --recreate-pods \
+      --timeout 3600 \
+      --wait \
+      "${HELM_PARAMS[@]}"
+
+  # Wait for UAA release
+  wait_for_release uaa
+fi
 
 # Deploy CF
-CA_CERT="$(get_internal_ca_cert)"
-
 set_helm_params # Resets HELM_PARAMS
 set_scf_sizing_params # Adds scf sizing params to HELM_PARAMS
 
 echo SCF customization ...
 echo "${HELM_PARAMS[@]}" | sed 's/kube\.registry\.password=[^[:space:]]*/kube.registry.password=<REDACTED>/g'
 
+if [ "${EMBEDDED_UAA:-false}" = "true" ]; then
```

Comment:
why not [[ == ]]?

Reply:
Fixed in bfa71748b534ee884718a00cb734a9dd21e3b621